.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+jobs:
+  build:
+    name: build
+    strategy:
+      matrix:
+        os: [macos-11]
+        python-version: [ 3.8 ]
+    runs-on: ${{ matrix.os }}
+    env:
+      CP_MYSQL_TEST_HOST: "127.0.0.1"
+      CP_MYSQL_TEST_DB: "cellprofiler_test"
+      CP_MYSQL_TEST_USER: "root"
+      CP_MYSQL_TEST_PASSWORD: "None"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          architecture: x64
+          python-version: ${{ matrix.python-version }}
+      - name: Get pip cache dir
+        id: pip-cache-dir
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+      - name: Cache pip
+        uses: actions/cache@v3
+        id: cache-pip
+        with:
+          path: ${{ steps.pip-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '14.0.1' # The JDK version to make available on the path.
+          java-package: jdk
+          architecture: x64
+      - name: Initial installs
+        run: |
+          pip install pyinstaller
+          pip install --upgrade pip setuptools wheel
+          pip install numpy>=1.20.1
+          git clone https://github.com/CellProfiler/python-javabridge.git ~/python-javabridge
+          git clone https://github.com/CellProfiler/python-bioformats.git ~/python-bioformats
+          git clone https://github.com/CellProfiler/centrosome.git ~/centrosome
+          git clone https://github.com/CellProfiler/core.git ~/core
+          # git clone https://github.com/CellProfiler/CellProfiler.git ~/cellprofiler
+      - name: Linux - install mysql and build tools
```

Comment:
If we decide against adding linux for now, do we wan to clean this up?

Reply:
Will do.